#Goal: Find task space velocities

#Inputs
float64[] q_pos

float64[] q_vel
---
#Outputs
float64 x
float64 y
float64 z
float64 wx
float64 wy
float64 wz